library/general/src/modules/Popup.rb

```diff
@@ -1029,6 +1033,21 @@ def Feedback(headline, message, &block)
       ClearFeedback()
     end
 
+    # Hides feedback during block execution.
+    # After block it is shown again. When exception is raised, then it is
+    # shown again.
+    # @see {Feedback}, {ShowFeedback} and {ClearFeedback} for details
+    # @param block block to execute
+    def SuppressFeedback(&block)
+      raise ArgumentError, "block must be supplied" unless block
+
+      return block.call unless @feedback_open # just call block if feedback is not open
+
+      ClearFeedback()
+      block.call
+      ShowFeedback(@feedback_headline, @feedback_message)
```

Comment:
There is a different return value, if a feedback is displayed it returns `block.call` value otherwise it returns `ShowFeedback()`  return value which is `nil`.

Reply:
well, no return value is documented, so it is void and no value is guaranteed.